greg/data/greg.conf

```diff
@@ -138,6 +138,16 @@ tag_artist = {podcasttitle}
 tag_title = {title}
 tag_genre = Podcast
 
+# You can also add a front-cover image to your podcast, if you wish, like so:
+#
+# front_cover = path/to/image/file
```

Comment:
```suggestion
# coverart = path/to/image/file
```

Reply:
There is also a `tag_picture` mentioned earlier but I don’t know what it does. Doesn’t seem to be a supported tag in eyeD3…